cmd/kflex/init/init.go

```diff
@@ -50,16 +50,6 @@ func Init(ctx context.Context, kubeconfig, version, buildDate string, domain, ex
 		util.PrintStatus("OpenShift cluster detected", done, &wg)
 	}
 	done <- true
-	util.PrintStatus("Installing shared backend DB...", done, &wg)
-	ensureSystemDB(ctx, isOCP)
-	done <- true
-
-	util.PrintStatus("Waiting for shared backend DB to become ready...", done, &wg)
```

Comment:
Since Postgres is only used by control planes of type k8s (by default with shared DB), just the fact that the kubeflex controller manager is up does not guarantee that the postgres DB is up. I think we should still have this check after the chart is installed, as well as the check for the controller.